operator/apis/platform/v1alpha1/securedcluster_types.go

```diff
@@ -390,6 +390,29 @@ func (l LocalScannerComponentPolicy) Pointer() *LocalScannerComponentPolicy {
 	return &l
 }
 
+// LocalScannerV4ComponentPolicy is a type for values of spec.scannerV4.scannerComponent
+// +kubebuilder:validation:Enum=Default;AutoSense;Disabled
+type LocalScannerV4ComponentPolicy string
+
+const (
+	// LocalScannerV4ComponentDefault means that local scanner V4 will use the default semantics
+	// to determine wether scannerV4 components should be used.
```

Comment:
typo: whether

Reply:
Today I learned something, I was always thinking wether is correct :o